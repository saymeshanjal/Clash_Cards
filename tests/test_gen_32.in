99
8 160 2
12 328 1
12 370 3
16 127 4
10 380 5
12 154 1
4 76 3
12 102 3
10 268 4
4 252 2
12 288 3
14 213 3
10 83 3
2 186 3
12 383 3
10 292 5
12 286 1
14 75 3
10 375 3
8 296 2
16 365 5
10 107 1
6 7 4
4 271 5
2 140 5
10 289 3
6 134 4
10 364 1
10 264 3
2 219 1
2 251 4
4 119 1
16 57 4
2 35 2
8 131 3
14 375 2
12 377 3
6 309 3
6 225 4
4 393 4
2 170 5
12 243 1
14 74 4
14 2 1
12 309 2
4 83 5
14 64 2
6 90 2
6 236 2
6 381 1
14 22 3
12 275 1
10 49 2
10 103 4
16 261 5
12 358 4
8 189 3
8 239 4
2 121 5
14 3 4
14 154 1
2 375 1
12 387 3
10 199 3
12 198 1
10 164 5
14 71 1
2 202 2
12 114 3
16 33 1
14 303 5
8 248 3
8 157 5
12 58 2
16 351 5
12 51 4
6 137 3
10 178 2
12 263 4
16 278 4
12 156 4
10 319 2
8 147 5
14 88 3
2 247 1
2 37 3
12 119 2
4 380 2
16 113 1
2 289 4
6 62 1
10 386 2
12 38 1
10 156 5
12 140 2
14 140 1
6 321 1
16 354 3
16 298 3
