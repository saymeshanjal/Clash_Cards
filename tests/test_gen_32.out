20 21 29
462 462 0
430 494 0
3809 5406 3655
45 207 0
462 462 0
1 5 0
430 494 0
44 133 75
1 2 3
430 494 0
1652 1780 0
110 142 0
0 2 0
430 494 0
45 207 0
462 462 0
1652 1780 0
110 142 0
20 21 29
4248 8622 0
126 126 0
0 18 2
0 6 0
0 2 0
110 142 0
1 14 5
126 126 0
110 142 0
1 1 0
0 2 0
3 3 0
536 7097 5237
0 1 1
27 43 0
1078 1079 1275
430 494 0
6 14 0
1 14 5
0 5 1
0 2 0
462 462 0
910 1520 1002
1716 1716 0
286 287 351
0 6 0
1078 1079 1275
5 6 9
5 6 9
10 10 0
532 2900 0
462 462 0
76 77 99
44 133 75
4248 8622 0
208 441 275
27 43 0
8 42 20
0 2 0
0 3102 330
1716 1716 0
1 1 0
430 494 0
110 142 0
462 462 0
45 207 0
1716 1716 0
0 1 1
430 494 0
6074 6796 0
987 2445 0
27 43 0
8 62 0
286 287 351
4248 8622 0
208 441 275
6 14 0
76 77 99
208 441 275
3809 5406 3655
208 441 275
76 77 99
8 62 0
1652 1780 0
1 1 0
0 2 0
286 287 351
1 2 3
6435 6435 0
0 2 0
10 10 0
76 77 99
461 463 0
45 207 0
286 287 351
1716 1716 0
10 10 0
6307 6563 0
6307 6563 0
